docs/arch/adr-009-storage-migration.md

```diff
@@ -15,24 +15,25 @@ Provide a migration path from OpenEBS to OpenEBS + Rook, triggered by the presen
 
 ## Solution
 
-A new field, "minimum node count", will be added to the rook config. (minimum value 2, requires openebs localpv to be present)
-When it is set, the rook operator will be installed on single node installs, but no ceph cluster will be created.
+A new field, `rook.minimumNodeCount`, will be added to the rook config.
+When set to a value of 2 or more (requires openebs localpv to be present), the rook operator will be installed on single node installs, but no ceph cluster will be created.
+When unset or set to a value of 1, the Rook storage class will function as at does today, backed by Rook-Ceph.
 
 An example spec is provided below:
 
 ```yaml
 spec:
   rook:
     version: "1.10.x"
-    storageClassName: "distributed"
+    storageClassName: "scaling"
```

Comment:
`scaling` isn't a name you choose (maybe we should put it in the ekco spec)

`distributed` is the name you get once rook has actually scaled up

```suggestion
    storageClassName: "distributed"
```